lib/manager/gitlabci/utils.ts

```diff
@@ -0,0 +1,13 @@
+// !reference [.setup, script]
```

Comment:
```suggestion

```

Do we need this comment to remind us to do something later? Or can this be removed?

Reply:
not sure, it's just a reminder about the format. I'll remove.